backend/src/Core/FurnitureShop.Core.Contracts/Mobile/Products/UpdateProduct.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using LeanCode.CQRS;
+using LeanCode.CQRS.Security;
+
+namespace FurnitureShop.Core.Contracts.Mobile.Products
+{
+    // [AuthorizeWhenHasAnyOf(Auth.Roles.User,Auth.Roles.Admin)]
+    [AllowUnauthorized]
+    public class UpdateProduct : IRemoteCommand
```

Comment:
U don't even need id to update the product?

Reply:
This is not a good idea to put the changed object like a single parameter. In general, u cant ask frontend about the whole product with id inside it. U should ask about id and some data for changes, or object, which represent data of this object

Reply:
Just separate id for object and needed data